linera-service/src/cli_wrappers/docker.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use super::util::get_github_root;
+use crate::util::CommandExt;
+use anyhow::Result;
+use pathdiff::diff_paths;
+use std::path::PathBuf;
+use tokio::process::Command;
+
+pub struct Docker;
+
+impl Docker {
+    pub async fn build(bin_path: &PathBuf, image_name: &str) -> Result<()> {
+        let github_root = get_github_root().await?;
```

Comment:
Can we take this path as input instead of guessing?

Reply:
Sure, yeah!